### ISSUE TITLE
New phase of the wither storm

### ISSUE THREAD

**Issue by Abasthepro** [nonamecrackers2/crackers-wither-storm-mod#106]
The wither storm when got severd and in his final phase if the player didn't kill it it will grow bigger than phase 7

**Comment by Coolthingiguess** [comment]
Back when the mod was just beginning, a made a very similar suggestion to this, adding a phase 8, however, it was declined because Minecraft is not meant to handle large entities, causing lag, so it would generally be unusable to most people

**Comment by nonamecrackers2** [comment]
Yes, however if I'm able to improve the model performances, adding more phases could be possible. I'll leave this open as a reminder.

**Comment by myron-obsolete-account** [comment]
I mean this kinda looks like a new phase to me
![image](https://github.com/nonamecrackers2/crackers-wither-storm-mod/assets/53058491/4244fe1b-2f9f-435f-ae5e-1059c39eccd7)
or is it just phase 4 with a few more tentacles?
[source](https://youtu.be/DH1ZiOvluHA?t=69)

**Comment by mythicplays** [comment]
The Phase 8 should be a bit bigger than Phase 7, the phase 9 is even bigger phase 7 but will be a bit bigger phase 8 and phase 10 the final phase, also each new phases should have more enderman pulling blocks animation like the old phase 5.5 after certain amount of consumedentities and phase 7.5, the phase 10 should be around 15M consumedentities, the withered symbionts should still continue spawning and phase 10 should start playing the "final boss theme", the phase 8-9 should continue playing the "bowels exposed theme", also the theme at phase 7 should be changed too

**Comment by meteorchikit-spec** [comment]
Better an epic one like in the fashion of wither storm v sides soundtrack and support for optimization mods For example, I can make an example of the phases :

<img width="1920" height="1200" alt="Image" src="https://github.com/user-attachments/assets/b339ab1b-9c08-4f4d-9861-63e771fef24c" />
